src/containers/BankApi/BankApiAccounts/index.tsx

```diff
@@ -18,25 +18,31 @@ const BankApiAccounts: FC = () => {
 
       <DocEndpoint endpoint="/accounts" method="GET" />
       <RequestResponseSnippet
```

Comment:
Should we add query parameters as well here? like @VincentHch added [here](https://github.com/thenewboston-developers/Website/pull/980/files)

Reply:
query params fixed with 9e1a895
I got signature error since I'm not sure how to generate valid one

Also, important questions:
Should we set count to 2 or to the value provided on the API?
And should I make my result as Vincent did {}, {}, ... {} or should we do it this way (with just 2 elements as I did)

Reply:
Your way is correct. We just need 2 records as mentioned in the descriptions. but that would mean that the next will have value like limit=2 but I don't mind it being anything since we just want to share the structure here. So, you are good in my review. let's see if the community has some other opinion on it 👍

Reply:
For the POST APIs, we need to set up projects locally or what we can do is set up a public test one for devs and share the signature keys so not everyone has to go through the hassle to setup. Thoughts? @velibor7 @VincentHch @sbalasa @DucPhamTV @Dragan @makaramkd

Reply:
To verify the POST request (for `/invalid_blocks`), I looked directly at the code.
Here is the invalid block sent by the validator to the bank : https://github.com/thenewboston-developers/Validator/blob/master/v1/tasks/confirmation_block_queue.py#L128

Reply:
Maybe for now we should stick with setting up the project locally as I can see we are the only one working on it.

Reply:
Of course setting up a local env to test is great but I'm still not able to setup successfully. Currently I'm using my bank which is joined the network to do POST/PATCH APIs.

Reply:
Okay, let's setup our nodes locally first. I see a channel already created named 'node-help' we can discuss there if someone hits a bump so others who want to set up can get help as well.